### PR TITLE
RSDK-8013: Add linting rules and Makefile

### PR DESCRIPTION
**Pull request by Emerald-Z**

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
+"""Utility file including methods for image decoding"""
+
 from typing import Union
-from viam.media.video import ViamImage, RawImage
 from PIL import Image
 from viam.logging import getLogger
 from viam.media.video import CameraMimeType
+from viam.media.video import ViamImage, RawImage
```
Comment (bhaney) [inline src/utils.py:7]: Is RawImage being used anywhere? I think it could probably be removed

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    else
+    echo "Installing venv on Linux"
+    sudo apt-get install -y python3-venv
+    fi
+fi
+if [ "$UNAME" = "Darwin" ]
+then
+echo "Installing venv on Darwin"
+brew install python3-venv
+fi
```
Comment (bhaney) [inline build.sh:17]: I don't think this a brew package that is available - should probably remove this

**Review by bhaney — CHANGES_REQUESTED** [review]
nice! Just two things to do here

File: src/utils.py (hunk 1 above)
Comment (Emerald-Z) [reply src/utils.py:7]: I don't believe that it is!

**Pull request merged**